<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1258</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="rozpocznij">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>260</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Rozpocznij</string>
    </property>
   </widget>
   <widget class="QLabel" name="doobleheader">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dooble</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jakgrac">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>450</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Jak grać?</string>
    </property>
   </widget>
   <widget class="CustomListWidget" name="leftlist">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>481</width>
      <height>461</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="umiescobiektylabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Umieść obiekty:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dodajdoA">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj zdjęcia</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generujpdf">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>560</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Generuj PDF!</string>
    </property>
   </widget>
   <widget class="CustomListWidget" name="rightlist">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>50</y>
      <width>481</width>
      <height>461</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="dodajdoB">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>530</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj zdjęcia</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Oraz ich odpowiedniki:</string>
    </property>
   </widget>
   <widget class="QSlider" name="ile_elem">
    <property name="geometry">
     <rect>
      <x>739</x>
      <y>630</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
    <property name="tickInterval">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="rozmiarlabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>580</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wpisz rozmiar do którego przycinane będą zdjęcia:</string>
    </property>
   </widget>
   <widget class="QLabel" name="elemlabel">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>580</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ustaw liczbę elementów na jednej karcie:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="dlugosc">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>640</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>0</height>
     </size>
    </property>
    <property name="minimum">
     <number>150</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="szerokosc">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>640</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>150</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QLabel" name="dllabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>610</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Długość:</string>
    </property>
   </widget>
   <widget class="QLabel" name="szerlabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>610</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Szerokość:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_obok">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>620</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;4&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">customlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
